File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/person/PersonSearchSuggestionProvider.java
package de.tum.in.tumcampusapp.component.tumui.person;

import android.content.SearchRecentSuggestionsProvider;

/**
 * Suggestion provider for {@link PersonsSearchActivity}
 */
public class PersonSearchSuggestionProvider extends SearchRecentSuggestionsProvider {
    public final static String AUTHORITY = "de.tum.in.tumcampusapp.component.tumui.person.PersonSearchSuggestionProvider";

    public PersonSearchSuggestionProvider() {
        setupSuggestions(AUTHORITY, 1);
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/person/PersonSearchSuggestionProvider.kt
package de.tum.`in`.tumcampusapp.component.tumui.person

import android.content.SearchRecentSuggestionsProvider

/**
 * Suggestion provider for [PersonSearchActivity]
 */
class PersonSearchSuggestionProvider : SearchRecentSuggestionsProvider() {
    init {
        setupSuggestions(AUTHORITY, 1)
    }

    companion object {
        const val AUTHORITY = "de.tum.in.tumcampusapp.component.tumui.person.PersonSearchSuggestionProvider"
    }
}